addons/digest/data/digest_tips_data.xml

```diff
@@ -9,100 +9,66 @@
             <field name="kpi_res_users_connected">True</field>
             <field name="kpi_mail_message_total">True</field>
         </record>
-    </data>
 
-    <data>
         <record id="digest_tip_digest_0" model="digest.tip">
-            <field name="name">Invite Team Members</field>
-            <field name="sequence">1</field>
-            <field name="group_id" ref="base.group_erp_manager"/>
+            <field name="name">Tip: Speed up your workflow with shortcuts</field>
+            <field name="sequence">800</field>
```

Comment:
Try to have a bit more logical sequences: 1, 10, 20, 30. We could say something like

  * 0-99 is for generic tips that should appear before other tips;
  * then steps of ten for each application (i.e. account is 101-9, project is 111-20, recruitment is 121-30, ...)
  * then 200+ for generic tips that should appear "after other tips".

That way everything is clean and easy to update if necessary.